### DISCUSSION TITLE
NvimTree does not seem to attach when changing dirs

### DISCUSSION THREAD

**Discussion by chris-dem** [nvim-tree/nvim-tree.lua#2852] [Q&A]
When I change dir using `Telescope` or with regular cd, I can't seem to use my keybindings. The commands are loaded but my bindings do not work. Only after I execute one of the commands I get my bindings back. I have tried the options but nothing: 
```
sync_root_with_cwd = true,
respect_buf_cwd = true,
reload_on_bufenter = true,
```

**Comment by alex-courtis** [comment]
Please raise a [bug report](https://github.com/nvim-tree/nvim-tree.lua/issues/new?assignees=&labels=bug&projects=&template=bug_report.yml) so that we may replicate and resolve.
